### ISSUE TITLE
Linux desktop app not working in background after using the "x" to send it to tray

### ISSUE THREAD

**Issue by cypherpunk1984** [tutao/tutanota#1955]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Describe the bug**
Linux desktop app v3.69.5
With freshly booted pc, app works perfectly in the background. After I open the app for the first time, clicking on a new email notification or opening it manually, and then using the "x" on the upper right corner to close it to tray, it shuts down, no more notifications, etc. I have to keep the window minimized on panel to keep it on.

**To Reproduce**
1. Reboot pc with "run on startup" checked
2. Open app (instantly opens no login needed)
3. Close/minimize app to tray using the "x" button on the upper right corner
4. Open app again (it starts on the login screen)

**Expected behavior**
App should go to tray when "x" button used. If I wanted closed I would use the newly added "quit" button on the left sidebar.

**Desktop (please complete the following information):**
 - OS: Linux Mint Tricia 19.3 (64 bits)
 - Desktop Environment: Cinnamon

**Additional context**
I know the tray icon doesn't work, and I know about the fix for that on 'System settings' => 'General' => 'Enable support for indicators'. Either way, I tried with that option on and off, and the behaviour is the same. After pressing the 'x' the app shuts down completely instead of going to tray and run in the background.

**Comment by charlag** [comment]
Hi
Do you have "Show Tray Icon" in App's Settings->Desktop checked?

We changed closing behavior, it now really closes the window instead of minimizing it. If you want to minimize it, you should use minimize button. But you should still receive notifications event when all windows are closed.

You could start it from the terminal (just run appimage) and check output when you close the window.

**Comment by cypherpunk1984** [comment]
> Hi
> Do you have "Show Tray Icon" in App's Settings->Desktop checked?
> 
> We changed closing behavior, it now really closes the window instead of minimizing it. If you want to minimize it, you should use minimize button. But you should still receive notifications event when all windows are closed.
> 
> You could start it from the terminal (just run appimage) and check output when you close the window.

Hi. Yes, I do have "show tray icon" enabled. Minimizing it, keeps it on, but its very annoying to have the window on the panel with other active windows that I really use (browser, etc.). I use Alt + Tab all the time to switch around windows, and having the tutanota app there is very annoying. If I use the 'x' the app completely shuts down, I don't recieve any notifications, nada.

**Comment by charlag** [comment]
Can you please run from the terminal and check if the app really stops when you close the window? Thanks.
Login behavior will improve when we will have offline and will be able to authenticate you in background.

**Comment by cypherpunk1984** [comment]
> Can you please run from the terminal and check if the app really stops when you close the window? Thanks.
> Login behavior will improve when we will have offline and will be able to authenticate you in background.

Yes, it's there, the process is there.
Sorry, I just checked sending an email from a different provider and they do show up. Last time I tried (a day or 2 days ago) they didn't, seems like the new update fixed that.
As you said, when I click on the notification it goes to the login screen instead of showing me the message, but that is alright. I can wait for that to improve as you said, I'm glad I can send it to background now. Thanks, and sorry for opening a new issue that wasn't actually an issue.
